src/sagemaker/feature_store/feature_group.py

```diff
@@ -665,6 +665,32 @@ def describe_feature_metadata(self, feature_name: str) -> Dict[str, Any]:
             feature_group_name=self.name, feature_name=feature_name
         )
 
+    def list_tags(self) -> Sequence[Dict[str, str]]:
+        """List all tags for a feature group.
+
+        Returns:
+            list of key, value pair of the tags.
+        """
+
+        feature_group_arn = self.sagemaker_session.describe_feature_group(
+            feature_group_name=self.name
+        ).get("FeatureGroupArn")
```

Comment:
Minor: might consider null pointer but I think it's unlikely to get a null with a successful call.

Reply:
I think this is unnecessary since FeatureGroupArn is a required field if describeFG returns without throwing errors.